core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -99,10 +99,11 @@ private[spark] object Utils extends Logging {
    * by setting the 'spark.debug.maxToStringFields' conf in SparkEnv.
    */
   val DEFAULT_MAX_TO_STRING_FIELDS = 25
+  val MAX_TO_STRING_FIELDS = "spark.debug.maxToStringFields"
```

Comment:
Can we move it to org.apache.spark.internal.config? I think our Core module should do the same thing like what we are doing in SQLConf.